playground/frontend/integration_test/common/common.dart

```diff
@@ -16,10 +16,17 @@
  * limitations under the License.
  */
 
+import 'dart:math';
+
 import 'package:flutter_test/flutter_test.dart';
 import 'package:playground/main.dart' as app;
 
 Future<void> init(WidgetTester wt) async {
   app.main();
   await wt.pumpAndSettle();
 }
+
+void expectSimilar(double a, double b) {
```

Comment:
No, I don't think that matcher could be instead of `expectSimilar`, because is different level of abstraction. `expect` method should get matcher as second argument, so it can't be a matcher itself

Reply:
```dart
Matcher closeToFraction(num value, double fraction) => closeTo(value, value * fraction);
Matcher onePerCentTolerance(num value) => closeToFraction(value, 0.01);
expect(a, onePerCentTolerance(b));
```
?